proc_stat.go

```diff
@@ -152,14 +152,35 @@ func (s ProcStat) StartTime() (float64, error) {
 	if err != nil {
 		return 0, err
 	}
-	return float64(stat.BootTime) + (float64(s.Starttime) / ticks()), nil
+	return float64(stat.BootTime) + (float64(s.Starttime) / clkTck), nil
 }
 
 // CPUTime returns the total CPU user and system time in seconds.
 func (s ProcStat) CPUTime() float64 {
-	return float64(s.UTime+s.STime) / ticks()
+	return float64(s.UTime+s.STime) / clkTck
+}
+
+// clkTck is the CLK_TCK value usually set to 100Hz. While this could be read
+// from uninstd.h without the need to call an external command, the cgo
+// dependency prevents cross-compilation among others. On all tested systems
+// supporting procfs, getconf was available and returned the expected value.
+var clkTck = 100.
+
+func init() {
+	output, err := exec.Command("getconf", "CLK_TCK").Output()
```

Comment:
There's security implications here, how do we know that 'getconf' that we're calling with all of our privileges is what we think it is?

This will also cause any go code that merely links to client_golang to do a fork+exec at startup, which may not be appropriate in all scenarios (think thousands of very short lived tasks talking to the pushgateway) and lead to questions from an strace.

We may be better off hardcoding, where have we seen that the value isn't 100?